Automattic/woocommerce-payments
# Send Store API URL to platform site

Fixes platform checkout issue 666

#### Changes proposed in this Pull Request
Retrieving the Store API namespace from the blocks of woocommerce core and sending this to the platform with other store data.

#### Testing instructions
- Install different versions of `woocommerce-blocks` on the merchant site and verify the URL on platform site
- From `woocommerce-blocks` >= 7.1.0 the value of `store_api_url` should be `wc/store/v1`
- For older versions of  `woocommerce-blocks` the value of `store_api_url` should be `wc/store`
